<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Настройки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Маска:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="inputMaskLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Путь к папке:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="inputFolderLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="selectFolderButton">
             <property name="text">
              <string>Выбрать папку</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Путь куда сохранить файл:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="outputPathLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Значение для модификации файла:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="modificationValueLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Переодичность (секунды):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="intervalSpinBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="deleteFilesCheckBox">
             <property name="text">
              <string>Удалить файл</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="overwriteCheckBox">
             <property name="text">
              <string>Перезаписать файл</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="periodicRunCheckBox">
             <property name="text">
              <string>Переодический опрос</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveSettingsButton">
         <property name="text">
          <string>Сохранить настройки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
